modules/README.md

```diff
@@ -60,7 +60,7 @@ This will create a new environment module in `/g/data/v10/public/modules/agdc-py
 
 Loading the module might conflict with other python modules you have loaded.
 
-The module will disable locally installed python packages to prevent conflicts by setting `PYTHONNOUSERSITE`
+The module will prevent conflicts with locally installed python packages by changing `PYTHONUSERBASE` for each release.
 
```

Comment:
Probably mention what it is being changed to?
Do we need to add it to PYTHONPATH too? In my setup I add to PYTHONPATH the equivalent of `~/.digitalearthau/${version}/local/lib/python3.6/site-packages`. And there's also the matter of adding `~/.digitalearthau/${version}/local/bin` to PATH.
And thanks Ben for this!

Reply:
No, don't need to explicitly add it to PYTHONPATH (instead the appropriate sub-directory will already appear in `sys.path` via a different mechanism), nor explicitly create the directories (pip handles that when necessary).

You're right about adding the `bin` to PATH.

Was initially inclined to keep the technical details out of the README, but perhaps explaining ("`pip install --user ...` will store things under `~/.digitalearthau`" is worthwhile..